shell/platform/windows/platform_handler_win32.cc

```diff
@@ -104,27 +105,25 @@ class ScopedClipboard {
   ScopedClipboard(ScopedClipboard const&) = delete;
   ScopedClipboard& operator=(ScopedClipboard const&) = delete;
 
-  // Attempts to open the clipboard for the given window, returning true if
-  // successful.
-  bool Open(HWND window);
+  // Attempts to open the clipboard for the given window, returning the error
+  // code in the case of failure and -1 otherwise.
```

Comment:
Why are you using -1 for success? That's pretty confusing, especially since `ERROR_SUCCESS` in Win32 is `0`.

Reply:
Yeah that was super confusing, switching to 0.